### PR TITLE
Updated all according to the view and updated CI as well

### PR DESCRIPTION
**Pull request by yuuteng**
I updated all the open issues according to the view and updated CI as well.

### CODE REVIEW

File: plugin/trino-snowflake/src/main/java/io/trino/plugin/snowflake/SnowflakeClientModule.java [hunk 1]
```diff
@@ -66,30 +63,6 @@ public ConnectionFactory getConnectionFactory(BaseJdbcConfig baseJdbcConfig, Sno
         properties.setProperty("TIME_OUTPUT_FORMAT", "HH24:MI:SS.FF9");
         snowflakeConfig.getTimestampNoTimezoneAsUTC().ifPresent(as_utc -> properties.setProperty("JDBC_TREAT_TIMESTAMP_NTZ_AS_UTC", as_utc ? "true" : "false"));
 
-        // Support for Corporate proxies
-        if (snowflakeConfig.getHTTPProxy().isPresent()) {
-            String proxy = snowflakeConfig.getHTTPProxy().get();
-
-            URL url = new URL(proxy);
```
Comment (dprophet) [inline plugin/trino-snowflake/src/main/java/io/trino/plugin/snowflake/SnowflakeClientModule.java]: Why was this removed?

Comment (yuuteng) [reply plugin/trino-snowflake/src/main/java/io/trino/plugin/snowflake/SnowflakeClientModule.java]: Because the reviewer said that these config properties are not tested and remove them from the initial PR.

Ref: https://github.com/trinodb/trino/pull/17909/files#r1230256442

Comment (yuuteng) [reply plugin/trino-snowflake/src/main/java/io/trino/plugin/snowflake/SnowflakeClientModule.java]: I reverted it back.

**Comment by yuuteng** [comment]
Closed it because it was merged at #14
